usage.md

```diff
@@ -8,45 +8,43 @@ Installation will be required for capabilities such as [self-provisioning](doc/d
 
 ### Requirements
 
-The first three requirements derive from a placeholder install script. These will be addressed in #121.
+1. ESXi/vCenter - the target virtualization environment
```

Comment:
Are there license constraints? Configuration limitations?
I'd suggest a note that it's only been tested with very simple vCenter configurations for now.